### PR TITLE
Add 3 columns to GUI

### PR DESCRIPTION
**Pull request by KohHanMing**
GUI has been refactored to show all lists in 3 separate columns.

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -22,35 +26,144 @@
...
+                              <Text fill="WHITE" strokeType="OUTSIDE" strokeWidth="0.0" text="Persons" wrappingWidth="80.287109375">
+                                 <font>
+                                    <Font name="Segoe UI Bold" size="22.0" />
+                                 </font>
+                              </Text>
+                              <HBox alignment="CENTER_RIGHT" prefHeight="34.0" prefWidth="239.0" HBox.hgrow="ALWAYS">
+                                 <children>
+                                    <ImageView fitHeight="33.0" fitWidth="33.0" pickOnBounds="true" preserveRatio="true">
+                                       <image>
+                                          <Image url="@../images/person-icon.png" />
```
Comment (hopinxian) [inline src/main/resources/view/MainWindow.fxml:90]: Can I check that these icons (person, location, visit and bg) are free art? Are there any references/acknowledgements that need to be made? If there is, copyright.txt would be the place to do it.

**Review by hopinxian — CHANGES_REQUESTED** [review]
GUI looks good! Would only like to confirm that the images used are free art.

File: src/main/resources/view/MainWindow.fxml (hunk 1 above)
Comment (KohHanMing) [reply src/main/resources/view/MainWindow.fxml:90]: Done, added copyright notes

**Review by hopinxian — APPROVED** [review]
LGTM

**Pull request merged**